scripts/data.py

```diff
@@ -49,7 +49,9 @@ def get_edge_tables(workspace: str) -> List[str]:
 
 def get_table_rows(workspace: str, table: str) -> List:
     """Return the rows of a table."""
-    resp = requests.get(f"{root_api_endpoint()}/workspaces/{workspace}/tables/{table}")
+    resp = requests.get(
+        f"{root_api_endpoint()}/workspaces/{workspace}/tables/{table}/rows"
```

Comment:
Was this endpoint just outdated before?

Reply:
Yeah, I made this change to the API as part of my proposed improvements, but I forgot to update your data uploader script.